superset/connectors/sqla/models.py

```diff
@@ -1450,7 +1450,7 @@ def get_from_clause(
         if not self.is_virtual:
             return self.get_sqla_table(), None
 
-        from_sql = self.get_rendered_sql(template_processor)
+        from_sql = self.get_rendered_sql(template_processor) + "\n"
```

Comment:
This is the fix compared to the original PR, basically forcing a line break after the FROM clause subquery. 
"""
```sql
-- BEFORE, notice how the `) AS` is in-comment
{...}
 FROM cte --COMMENT) AS virtual_table GROUP BY col1, col2

-- AFTER, notice the line break after the comment, allowing proper aliasing of the subquery
{...}
 FROM cte --COMMENT
) AS virtual_table GROUP BY col1, col2
```

Reply:
My only suggestion would be to add a comment in the code explaining why do we need the line break.

Reply:
@mistercrunch just to verify, does this logic work for `/* ... */` multiline comments?